crates/types/src/config/invocation.rs

```diff
@@ -39,14 +39,14 @@ pub struct InvocationOptions {
     #[serde(skip_serializing_if = "Option::is_none", default)]
     pub max_journal_retention: Option<FriendlyDuration>,
 
-    // TODO(slinkydeveloper) on v1.6 this option becomes mandatory, and serde should default to the values set below
     /// # Default retry policy
     ///
     /// The default retry policy to use for invocations.
     ///
     /// The retry policy can be customized on a service/handler basis, using the respective SDK APIs.
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub default_retry_policy: Option<InvocationRetryPolicyOptions>,
+    /// Check https://docs.restate.dev/services/configuration#retries for more details.
+    #[serde(default)]
+    pub default_retry_policy: InvocationRetryPolicyOptions,
```

Comment:
Are we ok with changing the behavior from killing to pausing invocations? This can mean for some users different operations since the paused invocations won't be automatically cleaned up unlike killed ones, right? This will now also happen to people that have the old retry policy configured and upgrade to a new Restate server version where this option is no longer used (although there was a deprecation warning printed).